package chap22_IO;

import java.io.FileNotFoundException;
import java.io.FileOutputStream;
import java.io.IOException;
import java.io.OutputStream;

public class _08_byteStreamToCharStream {

	public static void main(String[] args) {
		// TODO Auto-generated method stub

		
		
	}
	//try catch 는  exception에 대한 처리까지 가능한 기능
	// throws는 메소드를 호출한 부분에 에러가 나면 에러를 던지겠다. 알아서 처리해달라
	public static void write(String str) {
		try {
			OutputStream os = new FileOutputStream("C:/java/hp01.txt");
		} catch (FileNotFoundException e) {
			// TODO Auto-generated catch block
			e.printStackTrace();
		} catch(IOException ioe) {
			System.out.println(ioe.getMessage());
		} catch(Exception e) {
			System.out.println(e.getMessage());
		}
	}
	
	
}
